src/app/f2f/utils.js

```diff
@@ -0,0 +1,24 @@
+const moment = require("moment");
+
+/**
+ * formatDate takes a date (e.g. '1994-05-26') and the format it is in (e.g. 'YYYY-MM-DD)
```

Comment:
I can't see where this method is being used (outside of testing the method itself) Is it still needed?

Reply:
formatDate will be used in the ticket after this for the check answer page, I might have been a bit eager/in autopilot bringing it over in this branch so I'm happy to remove it and add it in again for the next merge if that's best?

Reply:
Nah don't worry about it. We all do it on occasion :)